### ISSUE TITLE
Support for \lt , \rt  macros in MathJax

### ISSUE THREAD

**Issue by KoyamaSohei** [sphinx-doc/sphinx#8517]
## Motivation

MathJax has `\lt` `\rt` macros but latex is not supported it.

When if `\lt` exists in :math: contents,

- `sphinx-build -b html` succeeded

- `sphinx-build -b latex`  succeeded but `latexmk` failed

Because `\lt` is Undefined control sequence.
```
! Undefined control sequence.
l.2213 \lt
           \mathsf{high}\), where \(k\) is an integer.
The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.
```

For examples: 
https://github.com/KoyamaSohei/optuna/blob/853d590b3a8b4f9b4613c0350a1c5b6d572cee5f/optuna/distributions.py#L193

## Why MathJax has `\lt` `\rt`  macros

http://docs.mathjax.org/en/latest/input/tex/html.html#html-special-characters

> In particular, that means that you have to be careful about things like less-than and greater-than signs (< and >), and ampersands (&), which have special meaning to web browsers. 

> Finally, there are \lt and \gt macros defined to make it easier to enter < and > using TeX-like syntax:

### Proposal 1
On building .tex file(`sphinx-build -b latex`), `\lt` , `\rt` convert to `<` ,`>`

### Proposal 2
Nothing to do ( Indicate in the document to NOT use `\lt` or `\rt`. )

Note: If `<`  exists in :math: contents, Sphinx convert to
- `&lt;` in `sphinx-build -b html`
- `<` in `sphinx-build -b latex`
So we need not using `\lt` , `\rt`

**Comment by tk0miya** [comment]
Hmm? Why do you want to use `\lt` and `\rt`? It seems MathJax also supports HTML entities (eg. `&lt;` and `&gt;`). Is there any reason not to use `<` and `>`?

**Comment by KoyamaSohei** [comment]
Yes, using `<` and `>` will solve the problem; there is no reason to use `\lt` ,`\rt`.
`Proposal 1` only gives support to MathJax `\lt` `\rt` users,who don't know that `\lt` `\rt` is not available in Sphinx.

**Comment by tk0miya** [comment]
The input format of the math directive has been defined as "LaTeX math syntax with support for Unicode symbols", not MathJax's. I feel there is no advantage to add a special syntax to the standard. Additionally, the change of syntax causes 3rd party math renderers not working (ex. https://pypi.org/project/sphinxcontrib-katex/). That's terrible.
https://docutils.sourceforge.io/docs/ref/rst/directives.html#math

-1 for changing.

**Comment by KoyamaSohei** [comment]
> The input format of the math directive has been defined as "LaTeX math syntax with support for Unicode symbols", not MathJax's

Thanks, I didn't know that.

**Comment by KoyamaSohei** [comment]
Sorry...

**Comment by tk0miya** [comment]
Don't mind. Thank you for your proposal :-)

**Comment by KoyamaSohei** [comment]
Thank you for taking the time :D
I'll take a break, because my concentration seems to be deteriorating.
